src/main/java/GameMonitor.java

```diff
@@ -1,10 +1,32 @@
-import com.fasterxml.jackson.databind.JsonNode;
-
 public class GameMonitor {
-    public void buttonPress(JsonNode clickPayload) {
-        String button = clickPayload.get("button").asText();
-        String actionType = clickPayload.get("actionType").asText();
 
-        System.out.println(String.format("Button %s sent the action %s", button, actionType));
+    Players firstPlayer;
+    Players secondPlayer;
+    Game spiel;
+
+    public GameMonitor () {
+        this.firstPlayer = new Players();
+        this.secondPlayer = new Players();
+        this.spiel = new Game(firstPlayer, secondPlayer);
```

Comment:
As we discussed last time, this should be a `match`.

Maybe to make it more clear (in regards to e.g. tennis):
* a `match` consists of `sets` and `games`
  * a `match` is won when a player wins the majority of `sets` (e.g. 2 out of 3, 3 out of 5, 5 out of 7)
* a `set` consists of `games`
  * in tennis you need to win 6 `games` to win a `set`
* a `game` consists of points
  * in tennis this means 4 points: `15, 30, 40, win`

<img width="655" alt="image" src="https://user-images.githubusercontent.com/1110551/33491151-a468b8b6-d6b9-11e7-8cd3-1b3cdb7765c6.png">

Now, in our case it's enough to have `match` and `games`. A `game` consists of 11 points. You have to win 2 games to win a match.